AnnoyancesFilter/Other/sections/self-promo.txt

```diff
@@ -159,6 +159,8 @@ www.kinopoisk.ru##div[class^="trailer-promo"]
 !
 ! SECTION: Self-promo - Regular rules
 !
+bogsfootwear.com##.weblayer--box-subscription-sms-email-signup--exit
```

Comment:
I don't see it there. Where did you get it?

Reply:
Same. But now I can't access to the site with VPN.

Reply:
reproduced.